readme.md

```diff
@@ -1,10 +1,24 @@
 # **Beetle**
-A beetle is tough, resilient, hard to kill, and rises above everyone else on an elytra
+_A collection of semicentralized utilities for Java development_
 
-An easy to use, powerful and modular utility system meant to be the replacement and killer for ElytraForce's old aUtils system.
+**please review this readme a248**
 
+An easy to use, powerful and modular utility system meant to be the replacement and killer for ElytraForce's old aUtils system.
 Beetle has a large focus on being object oriented and scalable.
 
+# What does it provide?
+
+Beetle is a decentralized software, so multiple applications will each have their own Beetle to work with (no sharing!)
+
+- A centralized source for safe sync/async runnable execution
```

Comment:
If Beetle is decentralized software, how is it that so many of its features are centralized?

Reply:
The software itself is decentralized (it's not like a single central beetle that api users will interact with, but each api user will have their own)